packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/sign-in-page/react-native/signin.md

```diff
@@ -0,0 +1,31 @@
+The async `signIn` method will automatically redirect users to your Okta organziation for authentication. It will emit an event once a user successfully signs in. Make sure your event listeners are mounted and unmounted. Note: on iOS there isn't a onCancelled event. If the sign in process is cancelled, onError will be triggered.
```

Comment:
* "Okta _organization_ "
* "_an_ `onCancelled` event"
* "`onError` will be triggered"

Notes should be formatted on new lines:

```
> **Note:**
```